pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -167,8 +168,19 @@ func manageOpenShiftControllerManagerDeployment_v311_00_to_latest(client appscli
 	if len(imagePullSpec) > 0 {
 		required.Spec.Template.Spec.Containers[0].Image = imagePullSpec
 	}
-	// TODO: translate high-level operator log levels currently in https://github.com/openshift/api/pull/138
-	required.Spec.Template.Spec.Containers[0].Args = append(required.Spec.Template.Spec.Containers[0].Args, fmt.Sprintf("-v=%d", 4))
+
+	level := 2
```

Comment:
Can't we have a default case instead of this?

Reply:
i'd still have to declare the var up front, so that seemed silly to me.